main.js

```diff
@@ -1 +1,60 @@
 import customers from './customers.js'
+
+
+
+let person = [{
+  thumbNail: 'https://randomuser.me/api/portraits/thumb/women/89.jpg',
+  name: "Pauline Hughes",
+  email: "pauline.hughes@example.com" ,
+  address: "3747 Elgin St, Roseville, IN 93281"
+ 
+}]
+
+function listCustomers(personArray) {
+  let customerList = document.querySelector("#customer-list");
+  console.log(customerList);
+
+  for (let customer of customers) {
+    let newCustomer = document.createElement("li");
+
+/
+    let customerImage = document.createElement("img")
```

Comment:
A next step would be to see if you can abstract these pieces out into small functions, each one that does one thing, like `addImage` or something like that. Then your main function would call all the little functions.